.github/workflows/check-lambda-js.yml

```diff
@@ -10,15 +10,16 @@ on:
       - main
       - development
       - ia-web-development
+      - ia-development
     paths:
-      - backend/compact-connect/lambdas/cloudfront-csp/**
+      - backend/compact-connect/lambdas/nodejs/**
 
```

Comment:
A few lighthouse performance issues (these have been around for a bit, and I put these down more as a starting point for discussion – not as definite "ToDo" items).

<img width="916" alt="image" src="https://github.com/user-attachments/assets/c3cbc66b-30ef-4f13-94b3-f0376b8c057b" />

Reply:
That sounds good to me! Let's make a ticket for Cloudfront compression and cache policy.

Reply:
I created #408 

cc @isabeleliassen